scripts/asm-installer/install_asm

```diff
@@ -2563,6 +2594,11 @@ scrape_managed_urls() {
 }
 
 install_managed_components() {
+  if [[ "${USE_MCP_CNI}" -eq 1 ]]; then
+    info "Installing CNI into the cluster..."
+    kubectl apply -f "${MANAGED_CNI}"
```

Comment:
MCP checks for *pods* with `k8s-app=istio-cni-node`. I think this introduces a race condition if pods are slow to start. Probably unlikely though..

But it may be more robust to add a MCP specific label (so we can distinguish from OSS install?) to the Daemonset (so its always there) and check that.

Or we could wait for the pods here?

Reply:
After 10 minutes of Googling I'm not convinced I know what the best thing to do here is, so whatever you think the best way is I'm happy to do. Is this something that needs to go into this PR to be safe, or can we merge as-is to start allowing staging environments to test CNI and add it later?